apps/api/src/app/subscribers/subscribers.controller.ts

```diff
@@ -362,7 +362,7 @@ export class SubscribersController {
       mark: { [MarkEnum.SEEN]: true },
     });
 
-    return await this.markMessageAsUsecase.execute(command);
+    return (await this.markMessageAsUsecase.execute(command))[0];
```

Comment:
Just a nit. Shouldn't be better to fix this inside of the usecase? I haven't found anywhere else where this usecase is being used than these 2 controllers you modified so it would make sense to me change it there. There was any problem to not be able to do that?

Reply:
Hey @p-fernandez  thanks for looking into it, i see that we use it in the subscribers controller on the route @Post('/:subscriberId/messages/markAs') and widget controller route   @Post('/messages/markAs'). IMO we can remove those `/seen` endpoints in the future because the `/markAs` is more generic in our case. LMKWYT.

Reply:
I just noticed the deprecated endpoints. I guess we still can't remove them so do the change I propose would break current implementations. We could duplicate the use case to have one different for mark one message and mark multiple messages and rewire the endpoints as needed, but that's too much work for this PR. 😕